lib/metrics/amplitude.js

```diff
@@ -85,6 +85,8 @@ const EMAIL_TYPES = {
   postRemoveSecondaryEmail: 'secondary_email',
   postVerifyEmail: 'registration',
   postVerifySecondaryEmail: 'secondary_email',
+  postConsumeRecoveryCodeEmail: 'recovery_code_email',
+  postNewRecoveryCodesEmail: 'recovery_code_email',
```

Comment:
I'd tweak these, I don't think they want the `_email` suffix. And maybe they should more generic too: `totp` or `2fa` perhaps?